src/libtiled/mapreader.h

```diff
@@ -54,7 +54,7 @@ class TILEDSHARED_EXPORT MapReader
 {
 public:
     MapReader();
-    ~MapReader();
+    virtual ~MapReader();
```

Comment:
While the `TmxMapReader` indeed derives from this class, it is never used with polymorphism. Still, I think this is a fine addition as well just to avoid running into possible issues later.